hack/generator/pkg/astmodel/package_definition.go

```diff
@@ -77,7 +77,7 @@ func emitFiles(filesToGenerate map[string][]TypeDefiner, outputDir string) error
 
 		err := genFile.SaveToFile(outputFile)
 		if err != nil {
-			return fmt.Errorf("error saving definitions to file '%v'(%w)", outputFile, err)
+			return errors.Wrapf(err, "error saving definitions to file %q", outputFile)
```

Comment:
This doesn't include anything about the nested error in the main message; won't that mean that someone debugging an issue has to explicitly dig for the piece of information they need?

To illustrate, I think this is better:
```
error saving definitions to file "demo.txt" (no space on disk)
```
than
```
error saving definitions to file "demo.txt"
```

Reply:
Same issue with your other changes, I think. (BTW - I didn't know about `%q`, that's nice.)

Reply:
`Wrapf` doesn't need a formatting marker, it always includes the details of the wrapped error - the wrapped message coming from that example will be `error saving definitions to file "demo.txt": no space on disk`.

Reply:
That works for me. :+1: